src/app/core/services/issue.service.ts

```diff
@@ -119,7 +121,9 @@ export class IssueService {
 
   createIssue(title: string, description: string, severity: string, type: string): Observable<Issue> {
     const labelsArray = [this.createLabel('severity', severity), this.createLabel('type', type)];
-    const hiddenData = new Map([['session', this.sessionId]]);
+    const appVersion = new ApplicationService(this.githubService).currentVersion;
+    const appInstance = ElectronService.isElectron() ? 'Desktop' : 'Web';
```

Comment:
Just a query, why not just inject `ElectronService` into `IssueService`'s constructor then we can leave the `isElectron()` function as an instanced function as opposed to a static one? I think performance wise the impact would be minimal.

Reply:
Yeah that is possible as well, I was just a little hesitant about pulling the whole `ElectronService` just to use `isElectron()`.
Not sure what is the best practice, but maybe I will inject `ElectronService` to adhere to the current patterns